ckanext/dcatapchharvest/profiles.py

```diff
@@ -717,6 +733,32 @@ def graph_from_dataset(self, dataset_dict, dataset_ref):  # noqa
                         Literal(reference_identifier)
                     ))
 
+        if dataset_dict.get("qualified_relations"):
```

Comment:
Short question: What is best practice in python: single or double question marks? Could we add this for linting?

Reply:
[There isn't a single accepted practise](https://pep8.org/#string-quotes) (although we should be consistent, which I realise this is not). When we upgrade to Python 3, we can start using [Black](https://github.com/psf/black) for linting, which requires double quotes.